src/coreclr/jit/emitarm64.cpp

```diff
@@ -943,6 +943,27 @@ void emitter::emitInsSanityCheck(instrDesc* id)
             assert(datasize == EA_8BYTE);
             break;
 
+        case IF_SVE_DO_2A: // ........xx...... .....X.MMMMddddd -- SVE saturating inc/dec register by predicate count
+            assert(isValidVectorDatasize(id->idOpSize())); // X
+
+            FALLTHROUGH;
+        case IF_SVE_DM_2A: // ........xx...... .......MMMMddddd -- SVE inc/dec register by predicate count
+            elemsize = id->idOpSize();
+            assert(insOptsNone(id->idInsOpt()));
+            assert(isGeneralRegister(id->idReg1()));   // ddddd
+            assert(isPredicateRegister(id->idReg2())); // MMMM
+            assert(isValidVectorElemsize(elemsize));   // xx
+            break;
+
+        case IF_SVE_DN_2A: // ........xx...... .......MMMMddddd -- SVE inc/dec vector by predicate count
+        case IF_SVE_DP_2A: // ........xx...... .......MMMMddddd -- SVE saturating inc/dec vector by predicate count
```

Comment:
Should these forms be merged?

(Or should we wait to decide on merging until after many/all instructions have been implemented?)

Reply:
I'm ok with either doing it now or waiting until after we've implemented all of them. @kunalspathak @TIHan do you have any preference for this?

Reply:
> waiting until after we've implemented all of them

Will be good to wait so we can track the coverage of them as listed in https://github.com/dotnet/runtime/issues/94549. We can then have one pass to combine all the duplicate ones.